docker-compose.yml

```diff
@@ -23,6 +23,8 @@ services:
       AWS_ACCESS_KEY_ID: ABC123
       AWS_SECRET_ACCESS_KEY: EFG456
       PUBSUB_EMULATOR_HOST: pubsub:8085
+      # Only presence of the environment variable is enforced
```

Comment:
The comment could be a hint clearer, it's otherwise a bit obscure for someone looking at the docker-compose file but not having read the code.

Reply:
updated comment